### ISSUE TITLE
Call to a member function getPackagePath() on null 

### ISSUE THREAD

**Issue by nilsbremke** [FriendsOfTYPO3/extension_builder#124]
After Installation Typo3 Extension Builder i get this massage: 

`Call to a member function getPackagePath() on null 
`
i don't know now what that means. 

Please Help... Or is it a probleme from extension... ??

**Comment by nicodh** [comment]
I have no clue since you did not provide any information about versions, stacktrace etc.

**Comment by stephangrass** [comment]
Maybe you named the extension folder other than "extension_builder"?

**Comment by awelvor** [comment]
I got the same message:

Call to a member function getPackagePath() on null


I am working with:  TYPO3 8.7.13
extension: example:

 'title' => 'Core Documentation Code Examples',
    'description' => 'This extension packages a number of code examples from the Core Documentation.',
    'category' => 'example',
    'author' => 'Documentation Team',
    'author_email' => 'documentation@typo3.org',
    'state' => 'alpha',


Please Help... Or is it a probleme from extension... ??

**Comment by awelvor** [comment]
Hi Nicodh

I got this alert when installing extension: examples
provided by:  https://github.com/TYPO3-extensions/examples


I did not use extension_builder

**Comment by nicodh** [comment]
This seems not to be extension_builder related...

**Comment by Pooja1790** [comment]
Oops, an error occurred!
Call to a member function getPackagePath() on null 
This is the error which i got while installing the mailchimp master extesion.
I am working on typo3 9.5 version. 
Please help.
Thanks in advance

**Comment by TinaLacatis** [comment]
Same problem!!!

I wanted to upload the extension but it comes this error too. I deleted the extension from my typo3conf/ext and created a new .ZIP with the name extension_builder (because when you download the extension from GitHub the name of document is extension_builder-master). In this .ZIP I put the extension files. After that I deleted the typo3temp/var/cash file and uploaded it again. And works :)
